requirements.txt

```diff
@@ -4,3 +4,4 @@ loguru
 datasets
 typer
 rich
+cffi
```

Comment:
Remove this

Reply:
In my testing, this seems required through the 'datasets' import:

```
conda create --name moa-test python=3.11
conda activate moa-test
pip install -r requirements.txt
DEBUG=1 python bot.py --model "groq/mixtral-8x7b-32768" --reference-models "groq/llama3-70b-8192" --reference-models "groq/mixtral-8x7b-32768" --reference-models "groq/gemma2-9b-it" --rounds 2
```

```
tijs@Perplexity MoA % DEBUG=1 python bot.py --model "groq/mixtral-8x7b-32768" --reference-models "groq/llama3-70b-8192" --reference-models "groq/mixtral-8x7b-32768" --reference-models "groq/gemma2-9b-it" --rounds 2  
Traceback (most recent call last):
  File "/Users/tijs/os/MoA/bot.py", line 1, in <module>
    import datasets
  File "/opt/homebrew/anaconda3/envs/moa-test/lib/python3.11/site-packages/datasets/__init__.py", line 17, in <module>
    from .arrow_dataset import Dataset
  File "/opt/homebrew/anaconda3/envs/moa-test/lib/python3.11/site-packages/datasets/arrow_dataset.py", line 75, in <module>
    from . import config
  File "/opt/homebrew/anaconda3/envs/moa-test/lib/python3.11/site-packages/datasets/config.py", line 145, in <module>
    importlib.import_module("soundfile").__libsndfile_version__
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/moa-test/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tijs/.local/lib/python3.11/site-packages/soundfile.py", line 17, in <module>
    from _soundfile import ffi as _ffi
  File "/Users/tijs/.local/lib/python3.11/site-packages/_soundfile.py", line 2, in <module>
    import _cffi_backend
ModuleNotFoundError: No module named '_cffi_backend'
(moa-test) 
```

Reply:
Actually, you're right. This looks like a bug in whatever version of 'datasets' I happened to pick up. I removed it.

So: Done!